torch/fx/experimental/proxy_tensor.py

```diff
@@ -243,17 +248,17 @@ def can_handle_tensor(x):
     if not pytree.tree_all_only(torch.Tensor, can_handle_tensor, (args, kwargs)):
         return NotImplemented
 
-    if func in CURRENT_DECOMPOSITION_TABLE:
+    if not external_call:
```

Comment:
We should not be decomposing a function if `external_call`, like `patched_bias` is. So how come we are decomposing `patched_bias` when `aten_ops=True` for `export`?

Reply:
It seems that the reason is that `dynamo` will ignore the wrapped functions and will attempt to inline them. The actual way to get around this is to make dynamo wrap aware, meaning that it will graph break on `wrap`.

Reply:
Oops, it seems that this is the `decomposed.code`:
```python
def forward(self, arg0_1):
    add = torch.ops.aten.add.Tensor(arg0_1, arg0_1);  arg0_1 = None
    _param_constant0 = self._param_constant0
    patched_bias = __main___patched_bias(_param_constant0, 4, 4);  _param_constant0 = None
    add_1 = torch.ops.aten.add.Tensor(add, patched_bias);  add = patched_bias = None
    return add_1
```
It seems that `__main___patched_bias` is not the same function as `patched_bias`... We may need to make some adjustments here...

Reply:
Similarly, the traced code for symbolic trace v.s. it's nodes are a little bit different... So this is not a bug? What's going on here?

```python
def forward(self, x):
    add = x + x;  x = None
    size = add.size(-2)
    size_1 = add.size(-1)
    bias = self.bias
    getitem = bias[(slice(None, None, None), slice(None, None, None), slice(0, size, None), slice(0, size_1, None))];  bias = size = size_1 = None
    add_1 = add + getitem;  add = getitem = None
    return add_1
    
opcode         name     target                       args                                                                                                      kwargs
-------------  -------  ---------------------------  --------------------------------------------------------------------------------------------------------  --------
placeholder    x        x                            ()                                                                                                        {}
call_function  add      <built-in function add>      (x, x)                                                                                                    {}
call_method    size     size                         (add, -2)                                                                                                 {}
call_method    size_1   size                         (add, -1)                                                                                                 {}
get_attr       bias     bias                         ()                                                                                                        {}
call_function  getitem  <built-in function getitem>  (bias, (slice(None, None, None), slice(None, None, None), slice(0, size, None), slice(0, size_1, None)))  {}
call_function  add_1    <built-in function add>      (add, getitem)                                                                                            {}
output         output   output                       (add_1,) 
```